platform_versioned_docs/version-24.3/studios/managing.mdx

```diff
@@ -49,84 +49,84 @@ To add a new data studio, complete the following steps:
           If you select the **Prebuilt container image** template, you cannot select **Install Conda packages** as these options are mutually exclusive.
           :::
 
-        - **Data studio name**
+        - **Studio name**
         - Optional: **Description**
      1. Select **Next**.
 
 1. For the **Summary** section, complete the following steps:
    1. Ensure that the specified configuration is correct.
    1. Save your configuration:
-      - If you want to save the data studio for future use, select **Add only**.
-      - If you want to save and immediately start the data studio, select **Add and start**.
+      - If you want to save the Studio for future use, select **Add only**.
+      - If you want to save and immediately start the Studio, select **Add and start**.
 
-You'll be returned to the Data Studios landing page that displays the list of data studio sessions in your workspace. You can inspect the configuration details of the session that you created by selecting the session name. The data studio that you created will be listed with a status of either **stopped** or **starting**, depending on whether you elected to only add the data studio or to start a session as well.
+You'll be returned to the Studios landing page that displays the list of Studio sessions in your workspace. You can inspect the configuration details of the session that you created by selecting the session name. The Studio that you created will be listed with a status of either **stopped** or **starting**, depending on whether you elected to only add it or to start a session as well.
 
 :::note
-By default, data studio sessions only have read permissions to mounted data paths. Write permissions can be added for specific cloud storage buckets during the compute environment configuration by defining additional **Allowed S3 Buckets**. This means that data can be written from the session back to the cloud storage path(s) mounted. To stop potential data loss, only one session per workspace can mount a unique data path. When adding a new session, data paths already mounted to other running sessions are unavailable. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the session immediately.
+By default, sessions only have read permissions to mounted data paths. Write permissions can be added for specific cloud storage buckets during the compute environment configuration by defining additional **Allowed S3 Buckets**. This means that data can be written from the session back to the cloud storage path(s) mounted. To stop potential data loss, only one session per workspace can mount a unique data path. When adding a new session, data paths already mounted to other running sessions are unavailable. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the session immediately.
```

Comment:
```suggestion
By default, Studio sessions only have read permissions to mounted data paths. Write permissions can be added for specific cloud storage buckets during the compute environment configuration by defining additional **Allowed S3 Buckets**. This means that data can be written from the session back to the cloud storage path(s) mounted. To stop potential data loss, only one session per workspace can mount a unique data path. When adding a new session, data paths already mounted to other running sessions are unavailable. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the session immediately.
```

Reply:
@robnewman, this one needs some feedback.

Reply:
Ah yes. This is updated. Change to:

```
By default, sessions only have read permissions to mounted data paths. Write permissions can be added
 for specific cloud storage buckets during the compute environment configuration by defining additional
 **Allowed S3 Buckets**. This means that data can be written from the session back to the cloud storage
 path(s) mounted. If a new file is uploaded to the cloud storage bucket path while a session is running, the
 file may not be available to the session immediately.
```